client-mobile/newOrg/Components/Home/Homeless/Homeless.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+var React = require('react-native');
+var {connect} = require('react-redux');
+
+var CreateHousehold = require('./dumb/CreateHousehold');
+var Invitations = require('./dumb/Invitations');
+var RoommateInvitations = require('./dumb/RoommateInvitations');
+
+var Homeless = React.createClass({
+  render() {
+    return this.props.invitations.length > 0 && this.props.house === null// user has pending invitations && user does not belong to a household (note: second condition is checked in Home.js too)
+      ? this.renderInvitations()
+      : this.renderCreateHousehold();
+  },
+
+  renderInvitations() {
+    return (
+      <Invitations
+        invitations={this.props.invitations}
+        submit={this.renderCreateHousehold}
```

Comment:
We want the rendered component to change based on state, not directly because of user actions. Submit should probably be a function that causes a change in the state